jobs/export_process/worker.go

```diff
@@ -39,27 +37,41 @@ func MakeWorker(jobServer *jobs.JobServer, app AppIface) model.Worker {
 
 		outPath := *app.Config().ExportSettings.Directory
 		ID := model.NewId()
-		app.SetID(ID)
-		exportFilename := ID + "_export.zip"
+		exportZipFilename := ID + "_export.zip"
+		exportLogFilename := ID + "_export.log"
 
-		rd, wr := io.Pipe()
+		zipReader, zipWriter := io.Pipe()
+		logReader, logWriter := io.Pipe()
+
+		errCh := make(chan *model.AppError, 2)
+		defer close(errCh)
+
+		go func() {
+			_, appErr := app.WriteFile(zipReader, filepath.Join(outPath, exportZipFilename))
+			errCh <- appErr
+		}()
 
-		errCh := make(chan *model.AppError, 1)
 		go func() {
-			defer close(errCh)
-			_, appErr := app.WriteFile(rd, filepath.Join(outPath, exportFilename))
+			_, appErr := app.WriteFile(logReader, filepath.Join(outPath, exportLogFilename))
```

Comment:
Why did you create two separate goroutines?

Reply:
because in single goroutine i was not able to create and write in more than one file, like here we have zip and log file